src/Data/CallbackRequest/Pisa/pisaCallbackRequestDataClass.ts

```diff
@@ -8,10 +8,18 @@ export async function pisaCallbackRequestDataClass() {
   // callback-request is more or less a "singleton". It only offers PUT, GET and DELETE.
   return provideDataClass('CallbackRequest', {
     connection: {
-      index: async () =>
-        ({
-          results: [await callbackRequestClient.get('')],
-        }) as DataIndexResponse,
+      index: async () => {
+        try {
+          return {
+            results: [await callbackRequestClient.get('')],
+          } as DataIndexResponse
+        } catch (error) {
+          if (!(error instanceof ClientError)) throw error
+          if (error.code !== 'record.not_found') throw error
+
+          return { results: [] }
```

Comment:
* [ ] FYI We could invert the conditions to make the fact that we're only handling `'record.not_found'` clearer to the reader. E.g. (untested):
```ts
if ((error instanceof ClientError) && error.code === 'record.not_found') return { results: [] }
throw error
```